frontend/src/js/standard-query-editor/QueryNodeActions.tsx

```diff
@@ -18,7 +18,7 @@ const StyledFaIcon = styled(FaIcon)`
 `;
 
 const StyledIconButton = styled(IconButton)`
-  padding: 0px 6px 4px;
+  padding: 3.5px 6px 0px;
```

Comment:
Hmm, how is this change related? Let's please avoid half-pixel values

Reply:
This was to fix 
![image](https://user-images.githubusercontent.com/118362236/205621353-fec510d5-1dea-4742-bf84-a4c70ed362f7.png)